git.md

```diff
@@ -195,6 +195,26 @@ Fix nasty little bug ISSUE-51 #resolve #time 2h30m
 * Keep the history *clean* and *simple*. Before pushing any branch to the
   public always ensure that it conforms to the style-guide. If it doesn't:
   squash, rebase, reword commits as necessary.
+  
+* When merging multiple small commits into a single change, make sure to
```

Comment:
I'm not sure about the word "merging", which for git has a very precise meaning. Do you mean "squashing", or do you mean "posting them for review in a single pull request containing multiple commits"?

Reply:
The second one. One a technical level, they can be merged, squashed, amended of rebased. I will change the wording.